dotnet/src/Connectors/Connectors.Memory.Postgres/PostgresConstants.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel.Connectors.Postgres;
+
+internal static class PostgresConstants
+{
+    /// <summary>A <see cref="HashSet{T}"/> of types that a key on the provided model may have.</summary>
+    public static readonly HashSet<Type> SupportedKeyTypes =
+    [
+        typeof(string),
```

Comment:
What about `Guid`? It seems to be supported in `PostgresVectorStore.GetCollection<TKey, TRecord>()`. 
`SupportedDataTypes` has the same problem.

Reply:
Good call, thanks for the catch. I'll set it as supported and add test coverage.